app/models/csv_export.rb

```diff
@@ -0,0 +1,57 @@
+class CsvExport < ActiveRecord::Base
+  belongs_to :user
+
+  def status?(state)
+    'ready' == state.to_s ? ['downloaded', 'finished'].include?(status) : status == state.to_s
+  end
+
+  def email
+    user.try :email
+  end
+  
+  def filename
+    "#{content}_#{formatted_create_at}.csv"
+  end
+
+  def full_filename
```

Comment:
might be nice to use different names like 'download_name' and `path` ... I was expecting full_filename to be `/export/#{filename}` ...